src/commands/__tests__/prepare.test.ts

```diff
@@ -52,4 +52,36 @@ describe('runPreReleaseCommand', () => {
       }
     );
   });
+
+  test('return true for valid version', () => {
```

Comment:
I would probably put these in a separate and/or nested describe block -- that way the test reads `checkVersionOrPart - returns true for valid version` rather than `runPreReleaseCommand - returns true for valid version`

Reply:
Good catch